src/components/ReplyComment/hooks/useReplyComment.js

```diff
@@ -13,6 +14,8 @@ const useReplyComment = ({itemProp, indexFeed, dataFeed, updateParent, page, get
   const [textComment, setTextComment] = React.useState('');
   const [newCommentList, setNewCommentList] = React.useState([]);
   const [item, setItem] = React.useState(itemProp);
+  const {params} = useRoute();
+  const [curHeight] = React.useState(Dimensions.get('window').height);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `params` from `useRoute()` and `curHeight` from `Dimensions.get('window').height` have been introduced. Make sure to handle potential null or undefined values for these variables to avoid runtime errors.

```diff
+ const {params} = useRoute();
+ const [curHeight] = React.useState(Dimensions.get('window').height);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->